examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -417,6 +417,12 @@ def parse_args(input_args=None):
         default=4,
         help=("The dimension of the LoRA update matrices."),
     )
+    parser.add_argument(
+        "--scale_scheduler",
+        action="store_true",
+        default=False,
+        help="Rescale Scheduler based on Zero SNR paper during training and sampling",
```

Comment:
there should probably be some mention about how terminal SNR requires a v_prediction model, as predicting epsilon results in no meaningful learning by the model at training time. this is stated in the original paper.

it feels like having a LoRA that can rescale betas on its own has zero value, we should be simply following the pretrained base model scheduler config.